test/Unit/MySqlInputBindingTests.cs

```diff
@@ -5,13 +5,13 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading;
-using static Microsoft.Azure.WebJobs.Extensions.MySql.MySqlConverters;
-using MySql.Data.MySqlClient;
+using Microsoft.Azure.WebJobs.Extensions.MySql.Tests.Common;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
 using Moq;
+using MySql.Data.MySqlClient;
 using Xunit;
-using Microsoft.Azure.WebJobs.Extensions.MySql.Tests.Common;
+using static Microsoft.Azure.WebJobs.Extensions.MySql.MySqlConverters;
```

Comment:
Do we add static for library import?

Reply:
Yes, the using static directive names a type whose static members and nested types you can access without specifying a type\class name. For details https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-directive#the-static-modifier

Chance of Impact: since this loc was there before this commit, here just order of sequence is change. Hence, considering no impact.